hooks/useFormFields.tsx

```diff
@@ -0,0 +1,14 @@
+import { useState } from 'react'
```

Comment:
does this need to be abstracted to a custom hook? it seems to be a pretty standard form filling function. could this live in a component or is there something i'm missing?